grafana/provisioning/dashboards/Boost Stats- updated.json

```diff
@@ -55,7 +55,7 @@
   "links": [],
   "panels": [
     {
-      "datasource": "DS_PROMETHEUS",
+      "datasource": "${DS_PROMETHEUS}",
```

Comment:
@DarkGhost7 could you change this everywhere to `DS_PROMETHEUS` again please ?
I think there will be provisioning issues with grafana variables otherwise.